packages/web/utils/helper.js

```diff
@@ -57,6 +57,26 @@ export const formatMoney = (value) => {
 	).replace(String.fromCharCode(160), String.fromCharCode(32));
 };
 
+/**
+ * Format currency to integer
+ *
+ * @param {string} value Currency formatted value (BRL or USD)
+ * @returns {number}
+ */
+export const formatCurrencyToInt = (value) => {
+	const BRL = /(?=.*\d)^(R\$\s)?(([1-9]\d{0,2}(\.\d{3})*)|0)?(,\d{1,2})?$/;
+	const numbersOnly = value.toString().replace(/[^\d.,]+/g, '');
+	let currencyAsInt = 0;
+
+	if (BRL.test(numbersOnly)) {
+		currencyAsInt = numbersOnly.replace(/\./g, '').replace(',', '.');
+	} else {
+		currencyAsInt = numbersOnly.replace(/,/g, '');
+	}
+
+	return parseFloat(currencyAsInt);
```

Comment:
It's easier (and maybe more effective) to remove all non-numeric characters from the value and then convert to number 👇 
 ```suggestion
		return Number(unMask(value));
```